features/user_can_comment.feature

```diff
@@ -0,0 +1,23 @@
+Feature:
+"As a visitor,
+In order to communicate with other users,
+I would like to be able to comment on their posts"
+
+  Background:
+    Given the following user exists
+    | name     |
+    | Fat Bob  |
+    | Slim Jim |
+    And the following post exists
+    | title        | text          | user    |
+    | Codeacademy  | Smooth-ride   | Fat Bob |
+    And I am logged in as "Slim Jim"
+    And I visit the site
+    And I click "FLORUM"
+
+  Scenario: User can comment a post
```

Comment:
I would add some sort of Sad Path scenario here

Reply:
Sad path added